client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -182,6 +184,14 @@ export const ApplicationDetailDrawer: React.FC<
               <TabTasksContent application={application} task={task} />
             </Tab>
           )}
+          {!application || !application.manifest ? null : (
+            <Tab
+              eventKey={TabKey.Manifest}
+              title={<TabTitleText>{t("terms.manifest")}</TabTitleText>}
+            >
+              <SchemaDefinedField jsonDocument={application.manifest} />
```

Comment:
Need to make sure the margins/padding around the component is consistent with the other tab contents